interface/patient_file/summary/add_edit_issue.php

```diff
@@ -25,9 +25,25 @@
 require_once($GLOBALS['srcdir'].'/options.inc.php');
 require_once($GLOBALS['fileroot'].'/custom/code_types.inc.php');
 require_once($GLOBALS['srcdir'].'/csv_like_join.php');
+
+// TBD - Resolve functional issues if opener is included in Header
```

Comment:
hi @mdsupport ,
Code looks good. Why can't we include the opener in the header call below?

Reply:
When it is included, somehow 'opener' value was getting clobbered.  That causes a reload to start pages - calendar and messages.  Guessing dialog.js checking and loading libraries but ran out of time.  Maybe someone else will have a simpler solution .. probably just modify the loading order.

In bootstrap world it will be better if popup component performs its actions and opener monitors popup window's unloading to respond.

Reply:
@sjpadgett, using code finder(required test step) may be the culprit.